<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1057</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sketching Application</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1057</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionClearSketch"/>
    <addaction name="actionOpenCGA"/>
    <addaction name="actionSaveGeometry"/>
    <addaction name="actionSketchyRendering"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Command</string>
    </property>
    <addaction name="actionPredict"/>
    <addaction name="actionFixGeometry"/>
   </widget>
   <widget class="QMenu" name="menuShape">
    <property name="title">
     <string>Shape</string>
    </property>
    <addaction name="actionAddBuildingMass"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTool"/>
   <addaction name="menuShape"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPredict">
   <property name="text">
    <string>Predict</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpenCGA">
   <property name="text">
    <string>Open CGA</string>
   </property>
  </action>
  <action name="actionCuboid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Cuboid</string>
   </property>
  </action>
  <action name="actionLShape">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>L-Shape</string>
   </property>
  </action>
  <action name="actionAddBuildingMass">
   <property name="text">
    <string>Add Building Mass</string>
   </property>
  </action>
  <action name="actionFixGeometry">
   <property name="text">
    <string>Fix</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionClearSketch">
   <property name="text">
    <string>Clear Sketch</string>
   </property>
  </action>
  <action name="actionSaveGeometry">
   <property name="text">
    <string>Save Geometry</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSketchyRendering">
   <property name="text">
    <string>Sketchy Rendering</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
